### DISCUSSION TITLE
Implement Auth.revokeToken method to support Sign in with Apple token revocation

### DISCUSSION THREAD

**Discussion by peterfriese** [firebase/flutterfire#10827] [Feature request]
As of [version 10.8.0](https://firebase.google.com/support/release-notes/ios#version_1080_-_april_11_2023), Firebase for Apple platforms support token revocation for Sign in with Apple (see the [documentation](https://firebase.google.com/docs/auth/ios/apple#token_revocation)).

Apps that offer account creation, must also provide users with a way to delete their account (and revoke the associated tokens) from within the app. 

To allow Flutter developers to use this API, it should be exposed (looks like [FLTFirebaseAuthPlugin.m](https://github.com/firebase/flutterfire/blob/master/packages/firebase_auth/firebase_auth/ios/Classes/FLTFirebaseAuthPlugin.m) might be a good place to put it).

See [this comment](https://github.com/firebase/firebase-ios-sdk/issues/9906#issuecomment-1518134941) on the Apple SDK repo for a person requesting this

**Comment by ChiaraMC** [comment]
Hey @peterfriese do you happen to know if there are updates on this?

**Comment by peterfriese** [comment]
I created a new issue for this: #11396

**Reply by ChiaraMC** [reply]
Thanks! Just so you know after I left my previous comment I noticed that there was an issue for this already open: https://github.com/firebase/flutterfire/issues/11065
